iron-overlay-behavior.html

```diff
@@ -293,7 +293,7 @@
 
     /**
      * Cancels the overlay.
-     * @param {?Event} event The original event
+     * @param {Event} [event] The original event
```

Comment:
Does `[event]` mean that `event` is an optional param?

I don't see anything about that syntax here: https://developers.google.com/closure/compiler/docs/js-for-compiler